### ISSUE TITLE
Use the High Contrast color mode in block screenshots

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/website-v2#385]
Most of the images around the website still use the original block colors, but we might want to change that. Scratch's new high contrast colors are more accessible (at least in terms of WCAG 2.0) and it's not an _extreme_ high contrast theme, so there's not really a reason not to use it, right?

This decision could also extend outside of the website, such as into YouTube videos or future "explanatory" pictures in the Scratch Addons settings page.

For instance, just look at this picture:
![when green flag clicked, wait 1 seconds](https://github.com/ScratchAddons/website-v2/assets/106490990/4211916a-c573-4cdb-a21d-584537f5c4e4)
The white text on the yellow Events blocks, and maybe on the gold Control blocks too, aren't that good. These are much better in high contrast. And if it seems legible, you're probably just used to it - if you think about it, it is probably not incredibly comfortable to look at.

For reference, the official Scratch Team YouTube channel uses high contrast, which makes sense - it's the official accessible theme, and isn't at an _extreme_ level.

Any thoughts? @ScratchAddons/admins @mxmou

I wanted to help settle this soon before my next blog post and future content elsewhere, just in case we choose the original colors for whatever reason. Ideally, everyone would use the color mode we decide on when contributing their screenshots, GIFs, and videos.

**Comment by WorldLanguages** [comment]
It would be hard to enforce this in the blog and the YouTube channel, but we could consider this for the addon screenshots that are displayed on scratchaddons.com/addons

**Comment by Hans5958** [comment]
Related #4

**Comment by Secret-chest** [comment]
I don't think it's important to read the blocks. They're older, most people use the original theme.

**Comment by BroJac5246** [comment]
I'm not against this, but I do want to point out that the older blocks are more recognizable (associated with Scratch).

**Comment by Hans5958** [comment]
This is a positive suggestion, but I do not wish this to be a strict requirement.

**Comment by mxmou** [comment]
> This is a positive suggestion, but I do not wish this to be a strict requirement.

I think addon screenshots should remain consistent - if we decide to use the high contrast mode, all of them should be updated. For blog posts and other content, a strict rule probably isn't needed, but I think using high contrast blocks makes sense.

**Comment by DNin01** [comment]
> I don't think it's important to read the blocks.

I guess the text on the blocks doesn't really matter. Are the blocks themselves easier to discern in high contrast?

**Comment by Secret-chest** [comment]
> Are the blocks themselves easier to discern in high contrast?

Actually, they have less contrast with the background, and the categories are harder to tell apart.

**Comment by Secret-chest** [comment]
> They have the same border color as normal blocks.

Which is very thin.
